tests/src/test/scala/common/rest/WskRestOperations.scala

```diff
@@ -1186,34 +1170,34 @@ class WskRestApi extends RunWskRestCmd with BaseApi {
                       operation: Option[String] = None,
                       expectedExitCode: Int = SUCCESS_EXIT,
                       cliCfgFile: Option[String] = None)(implicit wp: WskProps): RestResult = {
-    val parms = Map[String, JsValue]() ++ {
-      Map("basepath" -> basepathOrApiName.toJson)
-    } ++ {
+    val parms = Map("basepath" -> basepathOrApiName.toJson) ++ {
       relpath map { r =>
         Map("relpath" -> r.toJson)
-      } getOrElse Map[String, JsValue]()
+      } getOrElse Map.empty
     } ++ {
       operation map { o =>
         Map("operation" -> o.toJson)
-      } getOrElse Map[String, JsValue]()
+      } getOrElse Map.empty
     } ++ {
       Map("accesstoken" -> wp.authKey.toJson)
     } ++ {
       Map("spaceguid" -> wp.authKey.split(":")(0).toJson)
     }
 
-    val rr = invokeAction(
+    invokeAction(
       name = "apimgmt/deleteApi",
       parameters = parms,
       blocking = true,
       result = true,
       web = true,
       expectedExitCode = expectedExitCode)(wp)
-    return rr
   }
 }
 
-class RunWskRestCmd() extends FlatSpec with RunWskCmd with Matchers with ScalaFutures with WskActorSystem {
```

Comment:
that this is a class with an actor system that was extended by all the entity commands meant we allocated several instances of the actor system unnecessary. Further because this class isn't actually part of any test suite, the after all hook may not actually have been reached at all.